src/components/SendModal/ProvisionIdentity/ProvisionIdentityConfirm/ProvisionIdentityConfirm.js

```diff
@@ -103,31 +114,51 @@ class ProvisionIdentityConfirm extends Component {
           challenge_id: loginRequest.challenge.challenge_id,
           created_at: Number((Date.now() / 1000).toFixed(0)),
           name: identityName,
-          system_id: loginRequest.system_id,
-          parent: loginRequest.parent
+          system_id: systemid,
+          parent: parent
         }),
       });
 
-      const signedRequest = await signIdProvisioningRequest(coinObj, provisionRequest)
+      const signedRequest = await signIdProvisioningRequest(coinObj, provisionRequest);
 
       const res = await axios.post(
         webhookUrl,
         signedRequest
       );
 
-      await handleProvisioningResponse(coinObj, res.data, 5, async (address, fqn) => {
-        await linkVerusId(address, `${fqn}@`, coinObj.id);
-        await updateVerusIdWallet();
-        clearChainLifecycle(coinObj.id);
-        const setUserCoinsAction = setUserCoins(
-          this.props.activeCoinList,
-          this.props.activeAccount.id
-        )
-        this.props.dispatch(setUserCoinsAction);
-  
-        refreshActiveChainLifecycles(setUserCoinsAction.payload.activeCoinsForUser);  
-      })
-      submissionSuccess(res)
+      // store the provisioning request in encrypted memory
+      
+      const verified = await verifyIdProvisioningResponse(coinObj, res.data);
+      if (!verified) throw new Error('Failed to verify response from service');
+      
+      const confirmedProvisionedID = new primitives.LoginConsentProvisioningResponse(res.data)
+     
+      if (confirmedProvisionedID.decision.result.fully_qualified_name.split('.')[0].toLowerCase() !== identityName.split('.')[0].toLowerCase()
+            || confirmedProvisionedID.decision.result.parent !== parent) throw new Error('Response does not match the expected ID request');
+
+      const notification = {
```

Comment:
Notifications should be standardised, dispatched, edited, added, and removed using the functions provided in src/actions/actions/notifications/dispatchers/notifications.js, and the classes provided in src/utils/notification.js. This way, they can fit into the existing notification reducer, and be used for all sorts of non-verusid related purposes in the future. I would advise changing what you currently call the "SET_VERUSID_NOTIFICATIONS" action to a "SET_PENDING_VERUSIDS" action, and store specific information regarding the status, login request, info, etc. in there, and then when you need to show a notification on the home screen, or edit a notification to reflect a change, use dispatchAddNotification, or dispatchSetNotifications, and the relevant class for the desired notification type in src/utils/notification.js.

Different notification types display differently and can be used for different informtation, but there is no VerusID specific notification type.